pallets/test-token-provider/src/lib.rs

```diff
@@ -3,8 +3,8 @@
 #[cfg(test)]
 mod mock;
 
-#[cfg(test)]
-mod test;
+// #[cfg(test)]
+// mod test;
```

Comment:
why commenting this?

Reply:
This pallet is just meant for testing. We are going to remove it eventually.